Sources/JExtractSwift/Swift2JavaTranslator+Printing.swift

```diff
@@ -205,6 +205,11 @@ extension Swift2JavaTranslator {
       )
       printer.print("")
 
+      // Layout of the class
+      printClassMemoryLayout(&printer, decl)
+
+      printer.printParts("")
```

Comment:
```suggestion
      printer.print("")
```

should be enough to do some space